www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+* [Download](https://geocode.earth/data/whosonfirst/#IN) Who’s On First data in Shapefile and SQLite formats, via Geocode.Earth: 
+* [Learn](https://whosonfirst.org/blog/2023/05/31/shapefiles/) more about downloading Who’s On First data in various formats
+* Want the raw Karmashapes files? Full Karmashapes downloads are [available](https://karmashapes.wordpress.com/) on Justin’s blog. 
+
+# Next Steps
+
+Because the village and habitation data in India often overlaps, especially near the largest metro areas in the country, Karmashapes records were imported into Who’s On First most often as locality records and sometimes as neighbourhood records when they fell in an existing WOF locality. Practically speaking, this means many of the Who’s On First records in and around the ~1,500 largest metropolitan areas in India should be reviewed.
+
+Some of the input sources used to create Karmashapes use population estimates from before the last census and those vintage population values for localities and neighbourhoods may also need to be updated.
+
+[![](images/images/next-steps-1.png)]()
+![Next Steps](images/next-steps-1.png "Next Steps")
+_(above) Ahmedabad locality extent with curated neighbourhood boundaries and a few new, funky Karamashapes neighbourhoods on the margins_
+
+[![](images/images/next-steps-2.png)]()
+![Next Steps](images/next-steps-2.png "Next Steps")
+_(above) Delhi metropolitan area with pink locality polygons and blue neighbourhood polygons (original Quattroshapes neighbourhood polygons are contained in the urban core and Karmashapes neighbourhood polygons are visible on the margins)._
+
+So, reviewing the placetypes and geometries of records at the edge of large locality polygons like those shown in the above screenshots will be an important next step when updating WOF records.
+
+Improving neighborhood coverage within locality polygons and reviewing the hierarchies of all WOF records in India will also be a priority. In some cases, Karmashapes has created village and urban polygons that span slightly outside of expected parent region boundaries. We’ll want to review these polygons to ensure they’re extent is accurate and their hierarchies are expected.
+
+And, while Karmashapes provided English names for the villages, habitations, and urban areas of India, name localization into official and spoken languages for many places is a work in progress. While the most populous places that were already in WOF have good name localization coverage, thanks to Karmashapes over one million place records are now available in India and it is challenging to localize all place names into all local languages. We’re open to contributions if you have them!
+
+Lastly, if updates to PMGSY data become available (or any future dataset for localities in India are released under an open license), we’ll want to review those sources to keep WOF as up to date as possible - which it turns out can be quite tricky.
```

Comment:
Add new sentence to this graph: `We're also curious about updated population statistics from the long delayed 2021 Census.`